src/controllers/authControllers.js

```diff
@@ -16,7 +16,10 @@ const handleErrors = (err) => {
 
 
  
-
+  if (err.code === 11000) {
```

Comment:
As far as I know 11000 is a db/os error. And does this really occur in normal applications?

Reply:
Yes, bhaiya, error 11000 is for duplicate unique field error. For instance, since all emails here are supposed to be unique, if a person tries to create two accounts with the same email, he will face error 11000 while creating the second account.